MtfhReportingDataListener/SqsFunction.cs

```diff
@@ -80,17 +117,12 @@ private async Task ProcessMessageAsync(SQSEvent.SQSMessage message, ILambdaConte
             {
                 try
                 {
-                    IMessageProcessing processor = null;
-                    switch (entityEvent.EventType)
-                    {
-                        case EventTypes.TenureUpdatedEvent:
```

Comment:
Shouldn't this logic be maintained to allow for the listener to be extended to work with other types of events/data that needs to be synced with the data platform?

Reply:
I agree, but the reason why I removed it from here is because we have the same logic in UseCaseFactory.cs which is why I've called that method here instead. 

So in the future, when we need to add other events we can update the UseCaseFactory class

Reply:
Ah okay, thanks! This wasn't part of this PR so I missed that it's already in the UseCaseFactory class.